src/Symfony/Component/PropertyAccess/PropertyAccessor.php

```diff
@@ -75,21 +80,26 @@ class PropertyAccessor implements PropertyAccessorInterface
      * @param int $magicMethodsFlags A bitwise combination of the MAGIC_* constants
      *                               to specify the allowed magic methods (__get, __set, __call)
      *                               or self::DISALLOW_MAGIC_METHODS for none
-     * @param int $throw             A bitwise combination of the THROW_* constants
+     * @param int $throw A bitwise combination of the THROW_* constants
      *                               to specify when exceptions should be thrown
+     * @param int $bypassPropertyHooks A bitwise combination of the BYPASS_HOOKS_* constants
+     *                               to specify the hooks you want to bypass,
+     *                               or self::DO_NOT_BYPASS_HOOKS_ON_PROPERTY for none
      */
     public function __construct(
-        private int $magicMethodsFlags = self::MAGIC_GET | self::MAGIC_SET,
-        int $throw = self::THROW_ON_INVALID_PROPERTY_PATH,
-        ?CacheItemPoolInterface $cacheItemPool = null,
-        ?PropertyReadInfoExtractorInterface $readInfoExtractor = null,
+        private int                          $magicMethodsFlags = self::MAGIC_GET | self::MAGIC_SET,
+        int                                  $throw = self::THROW_ON_INVALID_PROPERTY_PATH,
+        ?CacheItemPoolInterface              $cacheItemPool = null,
+        ?PropertyReadInfoExtractorInterface  $readInfoExtractor = null,
         ?PropertyWriteInfoExtractorInterface $writeInfoExtractor = null,
+        int                                  $bypassPropertyHooks = self::DO_NOT_BYPASS_PROPERTY_HOOKS,
```

Comment:
Indentation changes should be reverted (same for comments)

Reply:
Indentation in args & comments reverted